README.md

```diff
@@ -100,6 +100,7 @@ To add a resource or contribute, please check out our contribution guide [here](
 
 - [CS50](https://cs50.harvard.edu/college/2020/fall/)
 - [Teach Yourself Computer Science](https://teachyourselfcs.com/)
+- [Algorithms and data structures in C/C++](https://www.cprogramming.com/algorithms-and-data-structures.html?inl=nv)
```

Comment:
Can you instead add this under C++?

Reply:
I wish I could but c++ don't seem to have a dedicated section and I am not well aware of the technology with which you have created different section so I can't create a section on my own. :<

Reply:
Oh sorry @NitinShaily I have mixed up PRs. someone else has an open PR with a new section C++ but it hasn't been merged yet due to some changes required from them.

Once it's merged I'll let you know so that you update your fork to the latest changes and add the link under C++. Thank you for your patience.

Reply:
ok  thanks.

Reply:
Hello @NitinShaily we have a C++ topic now. Please update your repository to the latest changes and add your resource under C++